test/runtime/ct.go

```diff
@@ -99,30 +97,40 @@ var _ = Describe("RuntimeDisabledValidatedConntrackTable", func() {
 
 		case helpers.Delete:
 			for _, x := range containersNames {
-				vm.ContainerRm(x).ExpectSuccess()
```

Comment:
Does this resolve flakes?

Reply:
I'm debugging an issue when the `AfterAll` is executed two times instead of one. In the second run, the system will fail because the container is not in place. In the AfterAll we want to have a clean state of containers, policies, etc.. An assertion is not needed at all ;-)

Reply:
Please link a GitHub issue that explains why you are removing this in the tests in a comment above this line. We should check for the success of commands returned to us by the Docker runtime. I recommend that we add logic within `ContainerRm` to check if the container is running before deleting it instead of just removing the check to see if the command succeeded. 

I specify this because @joestringer and I saw a spurious CI failure in which `docker run` failed due to a failure to perform `network.Join` correctly in the `cilium-docker` plugin and this went unnoticed by the tests because we don't check the return codes of Docker-related commands.

Reply:
@eloycoto GH issue is still not linked here - by linked, I mean inline in the code above the `ContainerRm` call itself.

Reply:
Sorry, it is indeed there...